jobs/release-microk8s/release-microk8s.groovy

```diff
@@ -11,13 +11,14 @@ def destroy_controller(controller) {
 
 pipeline {
     agent {
-        label "runner-${params.ARCH}"
+        label "runner-amd64-2"
```

Comment:
why limit this to `amd64-2`?  won't the [arm64 job](https://jenkins.canonical.com/k8s/job/release-microk8s-arch-arm64/) have problems if run on amd64?

If you *do* always want this on amd64, i'd pick one of the [`ps5-xx` workers](https://jenkins.canonical.com/k8s/computer/).  They're way faster.

Reply:
Good catch, thank you. This was a testing artifact, fixed.